### ISSUE TITLE
Inquiry about the signal-independent noise map calculated in the code and the setup of single image training

### ISSUE THREAD

**Issue by whtang906** [Reyhanehne/CVF-SID_PyTorch#3]
Hi, it's me again! I am a master's student and I am really interested in your work!
But I have found some questions about the technical detail.

According to figure 3a of the paper, you feed the signal-independent noise map <img src="https://render.githubusercontent.com/render/math?math=\hat{N_i}"> with signal-dependent noise map <img src="https://render.githubusercontent.com/render/math?math=\hat{N_d}"> into the network again and expect to get <img src="https://render.githubusercontent.com/render/math?math=[\hat{0},*,\hat{N_i}]">. I have some questions about this part.
1. Why we need to feed the signal-dependent noise map <img src="https://render.githubusercontent.com/render/math?math=\hat{N_d}"> as well? Why not just feed the signal-independent noise map <img src="https://render.githubusercontent.com/render/math?math=\hat{N_i}"> and expect to get <img src="https://render.githubusercontent.com/render/math?math=[\hat{0},\hat{0},\hat{N_i}]">.
2. According to the provided source code, it seems the signal-dependent noise map <img src="https://render.githubusercontent.com/render/math?math=\hat{N_d}"> wasn't fed into the network?
https://github.com/Reyhanehne/CVF-SID_PyTorch/blob/c89242a4ef5cc98481404192392a6f76e0fb0ec9/src/trainer/trainer.py#L87

Apart from this, can you share more detail about the setup of single image training? For example, the hyperparameters you have used (epochs, learning rate, etc) for the image shown in the appendix.

Thank you so much!

**Comment by myavartanoo** [comment]
Hi, thank you very much for your interest.
Since the signal-dependent noise map is multiplied by the zero clean image, it dose not have any effect but still there can be an unknown signal-dependent noise map (without any effect).
In other words the input to the network is 0+0^{\gamma}*N_d+N_i which is equal to N_i.

We use the same hyperparameters. We will provide the code and training details for training on the single image and other datasets very soon.

**Comment by whtang906** [comment]
Ah I see. I get your point now.

Good to hear that! Thank you so much for your effort.
